cfgov/unprocessed/css/organisms/expandable-group.less

```diff
@@ -43,6 +43,10 @@
     margin-bottom: unit(30px / @base-font-size-px, em);
 }
 
+.o-expandable-group .o-expandable {
+  border-bottom: none;
```

Comment:
Minor, indentation should be four spaces.

Reply:
Gah 😳 

Fixed.